### PR TITLE
Petting and Brushing functionality work

### PR DESCRIPTION
**Pull request by AlexHagan**

### CODE REVIEW

File: Assets/Scripts/CameraScript.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		cameraTransform = GetComponent<Transform>();
+        defaultCameraPosition = cameraTransform.position;
+		defaultCameraRotation = cameraTransform.rotation;
+    }
+	
+	public void Reset()
+	{
+		cameraTransform.position = defaultCameraPosition;
+		
+		cameraTransform.rotation = Quaternion.Slerp(cameraTransform.rotation, defaultCameraRotation, 0.5F);
```
Comment (kev-the-dev) [inline Assets/Scripts/CameraScript.cs]: Will this work? Isn't cameraTransform just a copy, not a reference to the camera's transform? I'm not seeing it ever reset

File: Assets/Scripts/Cat.cs [hunk 2]
```diff
@@ -281,9 +289,62 @@ void SelectTool(SelectedTool tool)
...
+	{
+		// When mouse released, act based on accumulated drag
+		is_drag = false;
+		double drag_time = Time.time - drag_start_time;
+		time_of_last_user_interaction = Time.time;
+
+ 		// A short drag is registered as a click, causing cat to begin user interaction behaviors
+		if (drag_time < 0.1) 
+		{
+			Camera.main.transform.LookAt(GetComponent<Transform>()); // main camera will follow cat
```
Comment (kev-the-dev) [inline Assets/Scripts/Cat.cs]: Isn't this also happening in the behavior tree? Maybe they're fighting

File: Assets/Scripts/CameraScript.cs (hunk 1 above)
Comment (AlexHagan) [reply Assets/Scripts/CameraScript.cs]: Just saw this-- cameraTransform is a reference to the camera's transform. I think the problem before was that I was accessing this Reset() function by using GetComponent<>() in CatBehaviorNodes.cs. However, that is a MonoBehaviour function, and none of the nodes inherit from MonoBehaviour, so for some reason it was silently failing.

I called the Reset() function inside Cat.cs and it worked. It should work as intended now in my latest commit!

**Review by kev-the-dev — APPROVED** [review]
Yay! It all seems to work in testing

**Pull request merged**